app/models/manageiq/providers/embedded_automation_manager/configuration_workflow.rb

```diff
@@ -1,2 +1,2 @@
-class ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationWorkflow < ManageIQ::Providers::AutomationManager::ConfigurationWorkflow
+class ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationWorkflow < ManageIQ::Providers::AutomationManager::ConfigurationScript
```

Comment:
@jameswnl please remove all `ConfigurationWorkflow` classes from `EmbeddedAutomationManager` and `EmbeddedAnsibleManger`. We currently don't support them.

Reply:
yes, in progress. removing this causing the spec to fail. Because embedded/external share the same set of specs and vcr cassettes. 
The vcr part is troublesome because of the `allow_unused_http_interactions => false` and embedded spec won't retrieve the `workflows` causing unused HTTP error.  Any suggestion?

Reply:
Shall we keep this in place?  As far as UI/Automate are not using them, no material effects (For embedded one, user cannot create a workflow anyway).

Otherwise, we'll have to split the specs and 95% of the specs would have to be duplicated code.

@Fryguy @blomquisg ?

Reply:
I'm fine with keeping it since it won't be used anyway.